robot/web/stream/server.go

```diff
@@ -507,11 +507,16 @@ func (server *Server) AddNewStreams(ctx context.Context) error {
 			server.logger.Warn("video streaming not supported on Windows yet")
 			break
 		}
+		framerate, err := server.getFramerateFromCamera(name)
```

Comment:
will this new logic break streaming for cameras that don't provide framerate in getProperties?

Reply:
Will add comment here -- encoder pipeline will default to 30fps (the same as the key-frame interval) if it cant pick up framerate.